twoliter/embedded/repack.Dockerfile

```diff
@@ -1,69 +0,0 @@
-# syntax=docker/dockerfile:1.4.3
```

Comment:
This commit seems to be missing the delete of repack.Dockerfile.dockerignore `twoliter: combine build and repack Dockerfiles`